### ISSUE TITLE
[BUG] Double text input on Windows

### ISSUE THREAD

**Issue by biase-d** [PotatoProject/Leaflet#66]
https://user-images.githubusercontent.com/47496233/125538033-157e26b3-3f4a-4be5-b834-9397211c3034.mp4

one key press from the keyboard is registering as multiple inputs at once

**Comment by dvd-bnc** [comment]
Yes got this several times, previously an issue within master. If it isn't fixed flutter side within releasing the newer version then I'll manually build with stable

**Comment by gargakshit** [comment]
> Yes got this several times, previously an issue within master. If it isn't fixed flutter side within releasing the newer version then I'll manually build with stable

Can't reproduce on stable
